src/components/Person.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Person({ person }) {
+  const wifeOrHusband = (person.sex === 'm' ? `wife's` : `husband's`);
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        My name is
+        {' '}
+        {person.name}
+      </h2>
+
+      {person.age && (
+      <p className="Person__age">
+        I am
+        {' '}
+        {person.age}
+      </p>
+      )}
```

Comment:
```suggestion
      {person.age && (
        <p className="Person__age">
          I am
          {' '}
          {person.age}
        </p>
      )}
```
Pay attention to the formatting